### PR TITLE
rename varaibles 'version' => 'release' and 'commit' => 'code'

### PR DESCRIPTION
**Pull request by mwarning**
this matches up with the (upcoming) openwrt json output changes and harmonizes variable names

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var config = {
   // Show help text for images
   showHelp: true,
   // Files to get data from
-  versions: {
+  releases: {
```
Comment (aparcar) [inline config.js]: How do you know it will be releases and not versions?

Comment (mwarning) [reply config.js]: yeah, I think version is better. I will change it back.

**Comment by mwarning** [comment]
@aparcar have look. I maybe should split up the commit more as well.

**Comment by mwarning** [comment]
To be honest, 'builds' instead of 'versions' is actually a better name. XD

**Pull request merged**